tests/test_project.py

```diff
@@ -2103,3 +2114,136 @@ def test_input_args(self):
                 assert len(tmp_project) == len(jobs)
                 # check that call does not fail:
                 tmp_project.detect_schema()
+
+
+class TestProjectStoreBase(test_h5store.TestH5StoreBase):
+
+    project_class = signac.Project
+
+    @pytest.fixture(autouse=True)
+    def setUp_base_h5Store(self, request):
+        self._tmp_dir = TemporaryDirectory(prefix='signac_')
+        request.addfinalizer(self._tmp_dir.cleanup)
+        self._tmp_pr = os.path.join(self._tmp_dir.name, 'pr')
+        self._tmp_wd = os.path.join(self._tmp_dir.name, 'wd')
+        os.mkdir(self._tmp_pr)
+        self.config = signac.common.config.load_config()
+        self.project = self.project_class.init_project(
+            name='testing_test_project',
+            root=self._tmp_pr,
+            workspace=self._tmp_wd)
+
+        warnings.filterwarnings('ignore', category=DeprecationWarning, module='signac')
+        self._fn_store = os.path.join(self._tmp_dir.name, 'signac_data.h5')
+        self._fn_store_other = os.path.join(self._tmp_dir.name, 'other.h5')
+
+    def get_h5store(self):
+        return self.project.data
+
+    @contextmanager
+    def open_h5store(self, **kwargs):
+        with self.get_h5store().open(**kwargs) as h5s:
+            yield h5s
+
+    def get_other_h5store(self):
+        return self.project.stores['other']
+
+    @contextmanager
+    def open_other_h5store(self, **kwargs):
+        with self.get_other_h5store().open(**kwargs) as h5s:
+            yield h5s
+
+
+class TestProjectStore(TestProjectStoreBase, test_h5store.TestH5Store):
+
+    """
+    This test opens multiple instances of H5Store, but
+    the project data interface opens one instance of H5Store.
+    Theses tests will (and should) fail using the project data interface.
+    """
+    def test_assign_valid_types_within_same_file(self):
+        pass
```

Comment:
just to make sure I understand correctly, the comments above are for this test function and they are overwritten with `pass` because they are expected to fail and thus should be skipped?

Reply:
Yes. The original tests opened multiple `H5Stores`. These tests use the `project.data` interface so we cannot test this.